src/BasisTheory.net/Tokens/Constants/TokenTypes.cs

```diff
@@ -5,5 +5,9 @@ public class TokenTypes
         public const string Token = "token";
```

Comment:
why is this not an enum?

Reply:
C# doesn't support string enums. You have to use something like Humanizer to make that work.

Reply:
gotcha, this is just used by the faker in the tests, i guess it's just to expose the list to consumers of the SDK. We don't want to force them to use Humanizer then?